friendly-captcha/includes/settings.php

```diff
@@ -587,13 +624,14 @@ function frcaptcha_settings_init() {
             'frcaptcha_widget_settings_section',
             array(
                 "option_name" => FriendlyCaptcha_Plugin::$option_widget_language_name,
-                "description" => "Set the language for the widget. Need another language? <a href=\"https://docs.friendlycaptcha.com/#/widget_api?id=data-lang-attribute\">Help us translate</a>.",
+                "description" => "Set the language for the widget. Need another language? <a href=\"https://docs.friendlycaptcha.com/#/widget_api?id=data-lang-attribute\">Help us translate</a>.<br />Only use this when Friendly Captcha fails to select the right language for your site.",
```

Comment:
> For v1 it does make sense to pick your own language (although maybe we could use __("en") as the default, and then through the Wordpress built-in localization stack it could select the correct language 🤔)

It does already use the Wordpress language for V1 when "automatic" is selected. So I think for most people it makes sense to use "automatic" in V1 as well.

I think the main difference between V1 and V2 is the number of languages available. For this we could have a separate "override" field for V2 which only shows the languages available in V2. @gzuidhof What do you think?

Reply:
I don't know how well WP does languages, but I'd hope it has the correct `lang="en-US"` at the top of the HTML doc (our own WP website has that).

I would say perhaps let's hide the setting entirely if v2 is enabled, and not set an explicit value at all?